packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -577,8 +587,10 @@ async function helmParameters(celoEnv: string) {
 }
 
 async function helmCommand(command: string) {
-  if (isCelotoolVerbose()) {
+  if (isCelotoolVerbose() && !command.includes(' dep build ')) {
```

Comment:
nice, this has been annoying

I take it the surrounding spaces aren't an issue?

Reply:
Should not be. The command is `helm dep build ...`. I tried to be as restrictive as possible but I guess there is no real case difference between with/out spaces.